backends-common/redis/src/main/java/org/apache/james/backends/redis/RedisHealthCheck.scala

```diff
@@ -30,11 +28,12 @@ import org.apache.james.core.healthcheck.{ComponentName, HealthCheck, Result}
 import org.reactivestreams.Publisher
 import reactor.core.scala.publisher.SMono
 
+import java.time.Duration
```

Comment:
Should not change the import order. We have an [Import layout suggestion](https://github.com/apache/james-project/blob/master/src/site/xdoc/server/dev-build.xml#L138) for James project :-)

But me I would not be too hard on this ^^.